spec/intl.html

```diff
@@ -1577,122 +1607,102 @@ <h1>Intl.DateTimeFormat.prototype.resolvedOptions ( )</h1>
               <th>Internal Slot</th>
```

Comment:
This table now has no `<del>` or `<ins>` markers any more. Can we remove it entirely and just let the links go to upstream ECMA-402?

Reply:
Done in a76057d7193d4b73acc10d2cd56feb452decedf1